src/main/blackbox/blackbox.c

```diff
@@ -1536,6 +1536,7 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE("gyro_notch_cutoff", "%d,%d",            gyroConfig()->gyro_soft_notch_cutoff_1,
                                                                             gyroConfig()->gyro_soft_notch_cutoff_2);
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_GYRO_TO_USE, "%d",            gyroConfig()->gyro_to_use);
+        BLACKBOX_PRINT_HEADER_LINE("gyro_debug_axis", "%d",                 gyro.gyroDebugAxis);
```

Comment:
```suggestion
        BLACKBOX_PRINT_HEADER_LINE("gyro_debug_axis", "%d",                 gyroConfig()->gyro_filter_debug_axis);

Reply:
> 

yea, I don't know why but that did not work properly. Let me try again to confirm

Reply:
> 

confirmed. that always results in a value of 1 no matter what debug axis is selected. I thought it might be because 'gyro_filter_debug_axis' is a string, but using %s just outputted garble

Reply:
Hmm. It is used same as in gyroInit() and introduced in https://github.com/betaflight/betaflight/commit/5bcf35c7af617a4c22ae727fe1a2def3e8d62a39

https://github.com/betaflight/betaflight/blob/14bcc13150e184f1709206147d3976a290c00dee/src/main/sensors/gyro_init.c#L615

Reply:
> Hmm. It is used same as in gyroInit() and introduced in [5bcf35c](https://github.com/betaflight/betaflight/commit/5bcf35c7af617a4c22ae727fe1a2def3e8d62a39)
> 
> https://github.com/betaflight/betaflight/blob/14bcc13150e184f1709206147d3976a290c00dee/src/main/sensors/gyro_init.c#L615

I think see the problem, missing #include "sensors/gyro_init.h"

Reply:
@haslinghuis I think it is fixed now. thanks. Tested on my end and was good